core/fileutil/dir_darwin.go

```diff
@@ -0,0 +1,12 @@
+//+build darwin
```

Comment:
TIL that naming the file `*_darwin.go` renders the `//+build darwin` tag redundant.